app/controllers/api/applications_controller.rb

```diff
@@ -50,6 +50,8 @@ def show
 
   def edit; end
 
+  # This will be used with
+  # /apiconfig/services/4/applications/new
```

Comment:
It's okay while WIP, but don't forget to remove before ready for review. This kind of comment can easily go outdated. Checking the Rails routes is usually a better way to relate controller actions and paths.